services/repository/src/main/java/org/sagebionetworks/auth/OpenIDConnectController.java

```diff
@@ -74,7 +74,7 @@ JsonWebKeySet getOIDCJsonWebKeySet() {
 	
 	/**
 	 * Create an OAuth 2.0 client.  Note:  After creating the client one must also set the client secret
-	 * and verify ownership of the host(s) in the registered redirect URIs.
+	 * and ask ACT to have their client verified.
```

Comment:
Maybe add link to https://docs.synapse.org/articles/using_synapse_as_an_oauth_server.html